xerparser/schemas/taskrsrc.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, account: ACCOUNT | None, resource: RSRC, **data: str) -> None
         self.target_end_date: datetime = datetime.strptime(
             data["target_end_date"], date_format
         )
-        self.target_lag_drtn_hr_cnt: int = int(data["target_lag_drtn_hr_cnt"])
+        self.target_lag_drtn_hr_cnt: float = float(data["target_lag_drtn_hr_cnt"])
```

Comment:
Change to:  `self.target_lag_drtn_hr_cnt: float = float(data["target_lag_drtn_hr_cnt"].replace(",", "."))`.  
Some countries use a coma rather than a dot in their floats.